generator_test.go

```diff
@@ -142,10 +156,53 @@ func testNewV1MissingNetworkFaultyRand(t *testing.T) {
 	}
 }
 
+func testNewV1Ordering(t *testing.T) {
+	u1 := Must(NewV1())
+	u2 := Must(NewV1())
+
+	n1, err := u1.UnixNanoOf()
+	if err != nil {
+		t.Fatal(err)
+	}
+	n2, err := u2.UnixNanoOf()
+	if err != nil {
+		t.Fatal(err)
+	}
+	if n1 > n2 {
+		t.Errorf("generated older UUID: %s > %s", u1, u2)
```

Comment:
Perhaps this message should print the nanoseconds, like the one below prints time.